models/claims_preprocessing/normalized_input/intermediate/normalized_input__int_eligibility_dates_normalize.sql

```diff
@@ -7,7 +7,11 @@
 
 select distinct
   elig.patient_id
-  , elig.patient_id||coalesce(elig.data_source,'')||coalesce(elig.payer,'')||coalesce(elig.plan,'')||coalesce(cast(elig.enrollment_start_date as {{ dbt.type_string() }}),'')||coalesce(cast(elig.enrollment_end_date as {{ dbt.type_string() }}),'') as patient_id_key
+  {% if target.type == 'fabric' %}
+    , elig.patient_id + coalesce(elig.data_source,'') + coalesce(elig.payer,'') + coalesce(elig."plan",'') + coalesce(cast(elig.enrollment_start_date as {{ dbt.type_string() }}),'') + coalesce(cast(elig.enrollment_end_date as {{ dbt.type_string() }}),'') as patient_id_key
```

Comment:
Should this be switched over to use the dbt concat macro as well? I saw you use that in other places.

Reply:
@sarah-tuva 
It -can- but still needs conditional logic anyway for the "plan" so I just left it.